src/main/java/com/recycle/buddy/model/output/RecognizeResponse.java

```diff
@@ -0,0 +1,24 @@
+package com.recycle.buddy.model.output;
+
+import java.util.List;
+
+public class RecognizeResponse {
+    private Error error;
+    private List<RecognitionResult> result;
+
+    public Error getError() {
+        return error;
+    }
+
+    public List<RecognitionResult> getResult() {
+        return result;
+    }
+
+    public void setError(Error error) {
```

Comment:
these setters are because we have to use a no-arg constructor for spring beans right?

Reply:
This is the model for Jackson library. It converts object to JSON and JSON to object and uses these methods.